### ISSUE TITLE
get_image() fails on m4a

### ISSUE THREAD

**Issue by azadkuh** [devsnd/tinytag#32]
as far as i've checked the `feature/mp4` has been merged back into `master`. the  `get_image()` fails on `m4a` files:
```python
tag = TinyTag.get('sample.m4a', image=True)
image_data = tag.get_image()
if not image_data:
    print('oops! this file has a cover image :-/')
```

**Comment by devsnd** [comment]
Ah yes, I forgot to finish the implemention of that one, but most of it is already in place, see https://github.com/devsnd/tinytag/blob/master/tinytag/tinytag.py#L287 (Removing the comment in this line should enable image parsing)

The problem here is that if I'd just enable it like that, the image will always be parsed and put into memory (the image *might* be huge). So I have to rewrite this part, so that only the location of the image will be recorded, but not the whole image be saved to ram.

**Comment by azadkuh** [comment]
I like to replace `mutagen` / `eyeD3` with `tinytag` in my repo: [cmus-osx](https://github.com/azadkuh/cmus-osx)

gr8, will try line:287.
thank you.

**Comment by devsnd** [comment]
Hey Amir,

I just [released version 0.16.0](https://pypi.python.org/pypi/tinytag) which comes with MP4 image support built in!

**Comment by azadkuh** [comment]
`tinytag` in: [cmus-osx](https://github.com/azadkuh/cmus-osx/releases/tag/v1.2.0)
